index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"/> 
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Version</title>
+</head>
+<body>
+  <nav>
+    <ul class="nav-bar">
+      <li class="logo">Logo</li>
+      <li><img class="hambug" src="images/Union.png" alt="hamburger"></li>
+    </ul>
+  </nav>
```

Comment:
Kindly surround this part with a `header` tag as it helps in SEO and Accessibility